crypto/client.go

```diff
@@ -15,8 +16,16 @@ func New(config Config) (Client, error) {
 		if config.Type != "" {
 			fmt.Println("Invalid encryption type, using default")
 		}
-		return NewAESOpenPGPClient(config)
-	case "aes-openpgp":
-		return NewAESOpenPGPClient(config)
+		fallthrough
+	case "aes_openpgp":
+		// Remove other settings
+		// TODO(leon): This is shitty.
+		config.Settings.AESGCMSettings = nil
```

Comment:
What if we nil out the irrelevant settings in [initConfig](https://github.com/leonklingele/pick/blob/443bb7b466fcea3ceff70d6a440d57eed2378aea/commands/config.go#L46)?

```go

func initConfig(configFile string) error {
    // sets default crypto settings...
    // loads a config if one is available...

    if config != nil {
        // nil unused settings here?
    }

	return nil
}
```